Tasks/DotNetCoreCLI/task.json

```diff
@@ -14,11 +14,10 @@
         "Agent"
     ],
     "demands": [],
-    "preview": true,
```

Comment:
This one is not required

Reply:
have added it back preview=true and reverted patch version update